### ISSUE TITLE
USB VCP not working

### ISSUE THREAD

**Issue by jflyper** [jflyper/hype7#1]
As commented in 6916c58, the USB device seems not activated at all; can't write to device register DCTL with gdb...

With F7 clearing `USB_OTG_DCTL_SDIS` from `DCTL` in `USB_DevConnect` immediately activate DP line. We can even do it manually by gdb.

**Comment by jflyper** [comment]
Further investigation revealed that the USB_OTG module is not even responding to core reset request. The following code in `USB_CoreReset` fails to detect `USB_OTG_GRSTCTL_CSRST ` bit cleared and returns `HAL_TIMEOUT`.
https://github.com/jflyper/hype7/blob/fca3a4e5269375e1ed0da6d6b607bc528004dcea/lib/main/STM32H7/Drivers/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_usb.c#L1097-L1110

The `USB_OTG_GRSTCTL_CSRST ` just doesn't go away...
```
(gdb) p USBx
$1 = (USB_OTG_GlobalTypeDef *) 0x40080000
(gdb) p (void *)USBx->GRSTCTL 
$2 = (void *) 0x80000001
(gdb) p USBx->GRSTCTL & USB_OTG_GRSTCTL_CSRST
$3 = 1
```

**Comment by jflyper** [comment]
Solved by #37
